site.yml.sample

```diff
@@ -517,7 +517,7 @@
             end: "{{ lookup('pipe', 'date +%Y%m%d%H%M%SZ') }}"
       when: dashboard_enabled | bool
 
-- hosts: '{{ groups["grafana-server"][0] | default(groups["mgrs"][0]) | default(groups["mons"][0]) | default(omit) }}'
+- hosts: grafana-server
```

Comment:
```suggestion
- hosts: "{{ grafana_server_group_name }}"
```

Reply:
that would introduce backward compatibility issues but I'm thinking about changing the name of this group